code/modules/mob/living/living.dm

```diff
@@ -854,6 +854,12 @@
 /mob/living/proc/get_visible_name()
 	return name
 
+/mob/living/can_put_in_hand(atom/I, hand_index)
+	. = ..()
+	if(!.)
+		return FALSE
+	return .
```

Comment:
This proc does nothing?

Reply:
oh, yeah, it's nothing. I think I added this for some reason, but it seems it's been removed lately.